docs/data/ingest-library/README.mdx

```diff
@@ -3,8 +3,26 @@ title: Ingest Library Introduction
 
-This section explains the key components of the /ingest library, particularly focusing on the prominent data types it exposes, the process of deriving changes from a ledger, and how tools like `transaction_reader` and `change_reader` can be used to read and extract transactions and events from the ledger.
+## Why use the Ingest Library?
+
+Applications can leverage the library to rapidly develop ingestion pipelines capable of acquiring real-time or historical Stellar network data and deriving custom data models. The library enables applications to traverse the hierarchal data structures of the network: [history archives](https://developers.stellar.org/docs/validators/admin-guide/environment-preparation#history-archives), [ledgers](https://developers.stellar.org/docs/learn/fundamentals/stellar-data-structures/ledgers), transactions, operations, ledger state changes, and events.
+
+Use the library for an intuitive, compile-time, type-safe developer experience to work with the main types of network data:
+
+### Ledger Entries
+
+Obtain the final state of all ledger entries on the network at close of any recent or historically aged checkpoint ledger sequence. A Checkpoint ledger occurs once every 64 ledgers, during which the network will publish this data to [history archives](https://developers.stellar.org/docs/validators/admin-guide/environment-preparation#history-archives) in the format of compressed files which contain lists of `BucketEntry`, wherein each contains one `LedgerEntry` and the `LedgerKey`.
+
+### Ledger Metadata
+
+Gain access to all transactions and their nested operations which were included in each closed ledger. It also contains the changes(pre/post) for each ledger entry when it is changed due to transaction activity and all [Soroban contract events](https://developers.stellar.org/docs/learn/encyclopedia/contract-development/events) emitted as a result of [contract](https://developers.stellar.org/docs/learn/fundamentals/stellar-data-structures/contracts) invocations during transaction execution. It is effectively a commit log for ledger entries.
+
+Ledger Metadata is the origin for any ingestion pipeline that wants to derive historical or current state models from stellar network. The library provides packages which clients can use to consume the metadata in streaming data format, with readers and callback functions.
+
+The streams of ledger metadata can be sourced in unbounded fashion as ongoing real-time ledgers close on the Stellar network, and also in a historical replay mode with a bounded range of past ledger sequences.
+
+![Ingest Library](/assets/ingest_lib.png) _\* Diagram: Depicts ingest library usage in real-time or historical network data acquistion from multiple sources (watcher core node as captive core, history archives, and CDP datastore)._
```

Comment:
This might warrant it's own section in the _Admin Guide_. I'm thinking similar to how the _Anchor Platform_ docs have an _Admin Guide_ -> _Architecture_ section (https://developers.stellar.org/platforms/anchor-platform/admin-guide/architecture). I can't comment directly on the png in the review, but I think we should work on this diagram a bit to make it less wordy/less busy. I think the rest of the content in this PR is solid though, so I might recommend either removing it and circling back on said "architecture" page later before we merge the feature branch. We could also just merge it now given it's a feature branch anyway, your call

Reply:
agreed, I'll attach this diagram to [Admin Guide/stellar-docs/1328](https://github.com/stellar/security/issues/520) for consideration. It's an [excalidraw](https://excalidraw.com/) and can re-open the image later in that ide and edit.